openbas-front/src/utils/hooks/useDataLoader.js

```diff
@@ -31,7 +33,10 @@ const useDataLoader = (loader = () => {}, refetchArg = []) => {
         sseConnect();
       }
     }, EVENT_TRY_DELAY);
-    sseClient.addEventListener('open', () => [...listeners.keys()].forEach((load) => load()));
+    sseClient.addEventListener('open', () => {
+      pristine = false;
+      [...listeners.keys()].forEach((load) => load());
+    });
```

Comment:
We need to add a pristine "state" here because at the start of the app (after login) useDataLoader() is called in index to initialize the stream, but as we have another useDataLoader in the sub pages the firsts requests given to useDataLoader is done twice